tools/spark.md

```diff
@@ -211,23 +213,22 @@ See also the
 [Longitudinal Tutorial](https://reports.telemetry.mozilla.org/post/tutorials/longitudinal_dataset.kp),
 one of the available example notebooks when you start a cluster.
 
-### Available Data Sources for SparkSQL
+### Available Data Sources for `SparkSQL`
 
-For information about available queryable data sources (e.g.  Longitudinal dataset),
-see
-[Choosing a Dataset](/concepts/choosing_a_dataset.md).
+For information about data sources available for querying (e.g. Longitudinal dataset),
+see [Choosing a Dataset](/concepts/choosing_a_dataset.md).
 
 These datasets are optimized for fast access, and will far out-perform
-analysis on the raw telemetry ping data.
+analysis on the raw Telemetry ping data.
 
 ### Accessing the Spark UI
 
-Go to https://localhost:8888/spark after ssh-ing into the spark cluster
+After establishing an SSH connection to the Spark cluster, go to https://localhost:8888/spark
```

Comment:
What a great revision. I didn't realize spell-checking would make us limit our use of jargon.